.github/workflows/check-lean4checker.yml

```diff
@@ -0,0 +1,47 @@
+name: build
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
```

Comment:
Honestly, it might be good to just enable this checker for every push and every PR. Depending on how people are developing, every push might be a bit much (something we could investigate later), but I think every PR would be good, regardless of target branch.

Reply:
Since the check only takes a minute to run, I like your idea of it running on every push/PR. Depending on the branch, whoever's working on it has the freedom to ignore the check.